test/spec.js

```diff
@@ -104,6 +105,20 @@ describe('Composable', function () {
         });
     });
 
+    describe('getTransformation', function () {
+        it('should allow an array instead of a string in transformation pipeline', function () {
+            var extractedData = Composable({
+                a: [
+                    ['window'],
+                    ['getProperties', 'testData2.stringWithComma'],
+                    ['split', ','],
+                    ['map', 'trim']
+                ]
+            });
+            expect(extractedData.a).toEqual(['Hello', 'and welcome!']);
+        })
+    });
```

Comment:
can we either add a test or modify the above to use mixed array and string transformations?  e.g.
```
var extractedData = Composable({
    a: [
        'window',
        'getProperties:testData2.stringWithComma',
        ['split', ','],
        'map:trim'
]
});
```

That should work, right?